### PR TITLE
Full catalan translation ca.json

### PR DESCRIPTION
**Pull request by marczinger**
All strings translated to catalan.

### CODE REVIEW

File: ca.json [hunk 1]
```diff
@@ -1,1990 +1,1990 @@
...
-		"msg-restart-required": "Changing this option requires a restart to take effect.",
-		"editor": {
+		"options": "Opcions",
+    	"plugin": "Connector",
+    	"builtin-plugins": "Connectors principals",
+    	"plugin-options": "Opcions del connector",
+    	"folder-path-example-placeholder": "Exemple: carpeta 1/carpeta 2",
+    	"file-path-example-placeholder": "Exemple: carpeta/nota",
+    	"msg-restart-required": "Canviar aquesta opció requereix un reinici perquè tingui efecte.",
+    	"editor": {
```
Comment (joethei) [inline ca.json:10]: Please fix the indentation here.

Comment (marczinger) [reply ca.json]: Corrected. I have done the commit again, is that correct? Sorry if I didn't have to do it.

**Comment by joethei** [comment]
The indentation is even worse than before, please fix it.

**Comment by marczinger** [comment]
How is that possible I copied the indentation on the official repo using tabs and all. I will check again sorry.

**Comment by marczinger** [comment]
The only thing I found was that 1 space indent on line 10, it was that? Sorry and thank you.

**Pull request merged**